portfolio/src/main/webapp/index.html

```diff
@@ -23,24 +23,52 @@ <h1>Hi! I'm Emily Zhao!</h1>
         <img width="400" src="/images/background.jpg">
     </figure> 
     <h2>This is my portfolio. Get ready to learn more about me!</h2>
-
-    <form action="/data" method="POST" style="text-align:center">
-      <p>Welcome to the sentence generator.</p>
-      <br/>
-      <p>Enter a verb (ends in -ing):</p>
-      <textarea name="verb">singing</textarea>
-      <br/><br/>
-      <p>Enter an adjective:</p>
-      <textarea name="adj">blue</textarea>
-      <br/><br/>
-      <p>Enter an animal:</p>
-      <textarea name="animal">rabbit</textarea>
-      <br/><br/>
-      <input type="submit" />
-    </form>
+    <div id="game" class="row">
+        <div class="column">
+            <div class="card">
+                <form action="/data" method="POST" style="text-align:center">
+                <p><b>Welcome to the sentence generator.</b></p>
+                <br/>
+                <p>Enter a verb (ends in -ing):</p>
+                <textarea name="verb" placeholder="Verb" cols="20" rows"2"></textarea>                
+                <br/><br/>
+                <p>Enter an adjective:</p>
+                <textarea name="adj" placeholder="Adjective" cols="20" rows"2"></textarea>
+                <br/><br/>
+                <p>Enter an animal:</p>
+                <textarea name="animal" placeholder="Animal" cols="20" rows"2"></textarea>
+                <br/><br/>
+                <input type="submit" name="count"/>
+                <br/><br/>
+                <label for="quantity">Choose a Quantity to Display:</label>
+                <select name="quantity" id="quantity" >
+                    <option value="1">1</option>
+                    <option value="2">2</option>
+                    <option value="3">3</option>
+                    <option value="4">4</option>
+                    <option value="5"selected="selected">5</option>
+                    <option value="6">6</option>
+                    <option value="7">7</option>
+                    <option value="8">8</option>
+                    <option value="9">9</option>
+                    <option value="10">10</option>
+                </select>  
+                <button type="Refresh" onclick="getSelect()">Refresh</button> 
```

Comment:
rather than triggering a refresh here, we can trigger the loadTasks() function to fetch the tasks with the requested #